package/MDAnalysis/analysis/rdf.py

```diff
@@ -645,9 +645,17 @@ def _single_frame(self):
                 backend=self.backend,
             )
 
-            for j, (idx1, idx2) in enumerate(pairs):
-                count, _ = np.histogram(dist[j], **self.rdf_settings)
-                self.results.count[i][idx1, idx2, :] += count
+            bins = self.rdf_settings["bins"]
+            minv, maxv = (
+                self.rdf_settings["range"][0],
+                self.rdf_settings["range"][1],
+            )
```

Comment:
We should be able to just use tuple assignment
```suggestion
           rmin, rmax = self.rdf_settings["range"]
```
I would also rename the variables. "v" does not relate to anything whereas "r" is really the distance in g(r).